app/styles/globals.css

```diff
@@ -76,29 +76,32 @@ i {
 }
 
 table {
+  line-height: 1.3;
   border-collapse: collapse;
 }
 
-th {
-  padding: 5px 0;
-  border-top: 1px solid var(--color-mono-gray-1);
-  border-bottom: 1px solid var(--color-mono-gray-1) !important;
+thead {
```

Comment:
This should ideally be in `Table.css`, but since there are tables in our webapp that don't the  `<Table /> ` component, this would have to do for now. I'm gonna fix later.